test/unit/client/creating_resource_with_meta_test.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:json_api/client.dart';
+import 'package:json_api/document.dart';
+import 'package:json_api/routing.dart';
+import 'package:json_api/server.dart';
+import 'package:test/test.dart';
+
+import '../../helper/expect_resources_equal.dart';
+
+void main() async {
+  final host = 'localhost';
+  final port = 80;
+  final base = Uri(scheme: 'http', host: host, port: port);
+  final routing = StandardRouting(base);
+
+  group('Client-generated ID', () {
```

Comment:
The description does not reflect the intention of the test.

Reply:
Get it!